compiler/damlc/daml-opts/daml-opts-types/DA/Daml/Options/Types.hs

```diff
@@ -70,6 +70,11 @@ data Options = Options
     -- ^ Whether to enable linting of the generated GHC Core. (Used in testing.)
   , optHaddock :: Haddock
     -- ^ Whether to enable lexer option `Opt_Haddock` (default is `Haddock False`).
+  , optCppPath :: Maybe FilePath
+    -- ^ Enable CPP, by giving filepath to the executable.
+  , optGhcVersionFile :: Maybe FilePath
```

Comment:
Do we care about being able to configure this?

Reply:
Answering my own question: No, this isn’t actually configurable, `mkOptions` will always overwrite it. Maybe we should error out in `mkOptions` if this is already set.

Reply:
I added a comment saying it's set by `mkOptions` at least